docs/installation.rst

```diff
@@ -0,0 +1,73 @@
+.. _installation:
+
+************
+Installation
+************
+
+Installing ``hvpy``
+-------------------
+
+If you are new to Python and/or do not have familiarity with `Python virtual environments <https://docs.python.org/3/tutorial/venv.html>`_, then we recommend starting by installing the `Anaconda Distribution <https://www.anaconda.com/distribution/>`_. This works on all platforms (linux, Mac, Windows) and installs a full-featured Python environment in your user directory without requiring root privileges.
+
+Using pip
+---------
+.. warning::
+
+    Users of the Anaconda Python distribution should follow the instructions
+    for :ref:`anaconda_install`.
+
+To install ``hvpy`` with `pip <https://pip.pypa.io/en/stable/>`_, run::
+
+    pip install hvpy
+
+.. _anaconda_install:
+
+Using Conda
+-----------
+
+To install ``hvpy`` using `conda <https://docs.conda.io/projects/conda/en/latest//>`_ run::
+
+    conda install hvpy
+
+Requirements
```

Comment:
I am not sure if this is useful information?

Reply:
Can you tell me what is missing? Or do we need this for a PyPI package?

Reply:
Yes, when you install a python package, it will install its dependencies